characters.txt

```diff
@@ -0,0 +1,26 @@
+Vanilla-visitor
+Andrew - Tax collector
+Steve - Groom
+Justin - Artisan
+Barry - Tinker
+
+Outlaws
+Evan - Stonemason
+Jon - Winemaker
+Mike - Blacksmith
+
+Andrew - Tax collector 
+Initial state
+- Outraged that you would delay him 
+
+1st talk option
+- Andrew bellows at you that he is a collector for the NTA (Nottingham Tax Association) and that you are costing the region valuable dollars by delaying him. If you continue, he demands recompense equal to his hourly wages.
```

Comment:
should this be the second option? I'd expect him to be civil when he first addresses the player.

Reply:
Yeah I hadn't considered his initial greeting. I'll make sure to consider that for all the characters moving forward.